src/osk.js

```diff
@@ -275,7 +277,11 @@ GuppyOSK.prototype.attach = function(guppy){
     add_matrix_control("-row", function(e){ e.preventDefault();guppy.engine.list_remove_row();guppy.render();});
 
     osk.appendChild(controls);
-    document.body.appendChild(osk);
+    if (target) {
```

Comment:
Could possibly simplify this to something like `(target || document.body).appendChild(osk);` ?